live-examples/js-examples/map-prototype-@@iterator.html

```diff
@@ -0,0 +1,15 @@
+<pre>
+<code id="static-js">var map1 = new Map();
+
+map1.set('0', 'foo');
+map1.set(1, 'bar');
+
+var iterator1 = map1[Symbol.iterator]();
+
+console.log(iterator1.next().value);
+// expected output: ["0", "foo"]
+
+console.log(iterator1.next().value);
```

Comment:
I made that same comment for the Set examples: consider using for...of instead of next().value?

```
var map1 = new Map();

map1.set('0', 'foo');
map1.set(1, 'bar');

var iterator1 = map1[Symbol.iterator]();

for (let item of iterator1) {
  console.log(item);
}
```

Reply:
OK. Cool.